### ISSUE TITLE
508-defect-3 [COGNITION] Direct Deposit Component - Previously entered data should be pre-populated when users enter edit in place mode

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#18437]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I want to have my previously entered values pre-populated into the edit in place form. It helps me make changes and prevents me hearing error messages announced immediately.

## Details

Screen readers will only hear error messages announced when users enter the edit-in-place UI unless we pre-populate users' previous values. Could we use the `oldState` state object to pre-populate values? Screenshots attached below.

## Acceptance Criteria

- [ ] Component should retain user's previously entered checking/savings radio button when entering / exiting edit in place
- [ ] Component should retain user's routing number when entering / exiting edit in place
- [ ] Component should retain user's account number when entering / exiting edit in place

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2021-01-12 at 4 08 20 PM](https://user-images.githubusercontent.com/934879/104380714-0c96e800-54f1-11eb-9778-834338d911b8.png)

---
![Screen Shot 2021-01-12 at 4 08 29 PM](https://user-images.githubusercontent.com/934879/104380730-115b9c00-54f1-11eb-8138-3a122b3e5a69.png)
